packages/flight-icons/package.json

```diff
@@ -18,8 +18,8 @@
   "scripts": {
     "typecheck": "pnpm tsc --noEmit",
     "lint": "pnpm eslint --quiet --ext .js,.ts,.tsx",
-    "sync": "ts-node --transpile-only ./scripts/sync",
-    "build": "ts-node --transpile-only ./scripts/build"
+    "sync": "ts-node --transpile-only ./scripts/sync.ts",
```

Comment:
@didoo I ran the sync/build manually to update the descriptions/keywords and it kept failing (couldn't find the sync script in the `/scripts` directory). I finally tried explicitly set the filename which ended up working, is this going to impact the automated pipeline? Otherwise, I'll just close this PR and run the workflow again.